Dockerfile

```diff
@@ -1,30 +1,10 @@
-FROM golang:1.14-alpine AS build
-LABEL maintainer="Sean Condon <sean@opennetworking.org>, Adib Rastegarnia <adib@opennetworking.org> "
-LABEL description="Builds a gNMI/gNOI simulator on a Debian distribution"
-
-RUN apk add --update bash openssl curl && rm -rf /var/cache/apk/*
-
-RUN apk update \
-    && apk add \
-      ca-certificates \
-    && apk update
-
-RUN apk add \
-    git \
-    iputils \
-    net-tools \
-    psmisc \
-    procps \
-    sudo
+ARG ONOS_BUILD_VERSION=undefined
 
+FROM onosproject/golang-build:$ONOS_BUILD_VERSION as build
 
 RUN mkdir -p $GOPATH \
     && GO111MODULE=on go get -u \
-      github.com/google/gnxi/gnmi_capabilities@6697a080bc2d3287d9614501a3298b3dcfea06df \
-      github.com/google/gnxi/gnmi_get@6697a080bc2d3287d9614501a3298b3dcfea06df \
-      github.com/google/gnxi/gnmi_set@6697a080bc2d3287d9614501a3298b3dcfea06df \
-      github.com/openconfig/gnmi/cmd/gnmi_cli@89b2bf29312cda887da916d0f3a32c1624b7935f \
-      github.com/google/gnxi/gnoi_target@6697a080bc2d3287d9614501a3298b3dcfea06df \ 
+      github.com/google/gnxi/gnoi_target@6697a080bc2d3287d9614501a3298b3dcfea06df \
       github.com/google/gnxi/gnoi_cert@6697a080bc2d3287d9614501a3298b3dcfea06df 
```

Comment:
we can remove gnoi_cert also if we want to remove gnoi_target

Reply:
I hadn't removed `gnoi_cert`, so leaving `gnoi_target` in there for the moment